registry/api/v2/descriptors.go

```diff
@@ -689,6 +689,14 @@ var routeDescriptors = []RouteDescriptor{
 									Format:      errorsBody,
 								},
 							},
+							{
+								Name:        "Not allowed",
+								Description: "Manifest put is not allowed because the registry is configured as a pull-through cache or for some other reason",
```

Comment:
I suppose I should edit these to mention the read-only mode in that PR once this is merged, right?